guides/V1_Start_Meeting.md

```diff
@@ -0,0 +1,55 @@
+---
+copyright: 'Copyright IBM Corp. 2018'
+link: 'start-meeting'
+is: 'experimental'
+---
+
+# Start a meeting within a space
+
+## Concepts
+
+The _startMeeting_ mutation allows the API caller to start a meeting with a **space**.  The mutation accepts a Space ID (spaceId) as an argument, and makes the request on behalf of the caller, starting a meeting within the specified space.  The result will reflect the whether the call was accepted for processing by the service.
```

Comment:
can the JWT used on this API call be that of an appID as identity and as client or does it expect a person as the identity?

Reply:
Good question.  The back-end implementation doesn't care provided the identity represented by the JWT token has the proper permission: `meeting_host`  I don't know if an app can be assigned that permission, or if there are gaps around that.